about.css

```diff
@@ -1 +1,460 @@
 @import url(root.css);
+
+/* *{
+    outline: red 1px solid !important;
+  } */
+
```

Comment:
Kindly take away every commented piece of code all through your project as it makes your codebase look a bit messy🥲